tests/ecp5/macc.ys

```diff
@@ -0,0 +1,15 @@
+read_verilog macc.v
+proc
+hierarchy -top top
+#Failed because of 14 unproven cells.
+equiv_opt -assert -map +/ecp5/cells_sim.v synth_ecp5 # equivalency check
+#equiv_opt -map +/ecp5/cells_sim.v synth_ecp5 # equivalency check
```

Comment:
Seams that now MULT18X18D is properly generated, but we do not have simulation models yet.
Adding something like "#Blocked by issue #1358 (Missing ECP5 simulation models)" will be fine for now I guess